packages/connect/src/api/ethereum/ethereumDefinitions.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'cross-fetch';
+
+import { EthereumDefinitions } from '@trezor/transport/lib/types/messages';
+
+// considerations:
+// - use request module, just like when downloading firmwares?
+// - caching?
+// - whitelist of baked ins? for example chain_id 1 will always be there. but this whitelist will change from version to version, not sure if
+//   it is worth maintaining
+// - offline use in suite desktop? seeing unknown token is probably enough?
```

Comment:
It might be weird for the users, so maybe some explanation in suite would be nice, but I am not sure.